spec/mongo/auth/scram_spec.rb

```diff
@@ -43,15 +43,16 @@
         Mongo::Auth::User.new(
           database: 'driver',
           user: 'notauser',
-          password: 'password'
+          password: 'password',
+          auth_mech: 'SCRAM-SHA-1'
         )
       end
 
       let(:cr) do
         described_class.new(user)
       end
 
-      it 'raises an exception', if: list_command_enabled? do
+      it 'raises an exception', if: scram_sha_1_enabled? do
```

Comment:
`scram_sha_1_enabled?` is defined as an alias to `list_command_enabled?` (because they were added in the same server version), so it seemed reasonable to just use it explicitly here to avoid confusion for future readers .